src/main/groovy/de/gliderpilot/gradle/semanticrelease/SemanticReleaseChangeLogService.groovy

```diff
@@ -237,6 +242,7 @@ class SemanticReleaseChangeLogService {
 
         Release release = repo.releases().create(tag)
         new Release.Smart(release).body(changeLog(commits(Version.valueOf(version.previousVersion)), version).toString())
+        releaseAsset(release.assets(), "$version.version")
```

Comment:
Or we could define a member releaseAssets of type FileCollection and create a method `releaseAssets(Object)`:

```
FileCollection releaseAssets = project.files()

def releaseAssets(Object... assets) {
    releaseAssets += project.files(assets)
}
```

The above would require some more changes, because the ChangelogService does not know the project yet. But it would allow the user of the plugin to just write

```
task sourceJar ...
task javadocZip ...
semanticRelease {
    changeLog {
        releaseAssets(jar, sourceJar, javadocZip)
    }
}
```

because project.files can handle Task instances and automatically uses the task outputs (https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:files(java.lang.Object[]))